### ISSUE TITLE
`aria-input-field-label` example bug? 

### ISSUE THREAD

**Issue by aminahbl** [dequelabs/axe-core#4288]
### Product

axe-core

### Description

There's an error in the "Correct markup solution" section of the https://dequeuniversity.com/rules/axe/4.8/aria-input-field-name page `select` example, which is given as:

```html
<!-- Select a color: -->
<div id="pass2" role="listbox" aria-labelledby="pass2Label">
    <div role="option">Orange</div>
</div>
 ```

However, it should be as is given here:

https://github.com/dequelabs/axe-core/blob/3ce3e3d0b7e70c0e9351254c7cf6bfa3a08074e2/test/integration/rules/aria-input-field-name/aria-input-field-name.html#L19-L23

Hope this helps. Thanks for all the brilliant work! :pray:

**Comment by straker** [comment]
Thanks for the issue. I see what you mean and we'll work on getting that fixed.

**Comment by CBID2** [comment]
Can I work on this?

**Comment by straker** [comment]
Change has been made and will be updated the next time the docs are built.
